addons/website_project_issue/models/project_issue.py

```diff
@@ -1,13 +1,29 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
-from odoo import api, exceptions, models, _
+from odoo import api, fields, exceptions, models
 
 
 class Issue(models.Model):
     _name = "project.issue"
     _inherit = ['project.issue']
 
+    main_attachment_ids = fields.One2many('ir.attachment', compute='_compute_attachment_ids', string="Main Attachments",
```

Comment:
I still don't understand the purpose of that field. Attachments of an issue should contain all attachments, including those from messages. Actually most attachments will come from messages, so even the name "main attachment" is not correct in my opinion.